Src/MoneyFox.Shared/ViewModels/ModifyPaymentViewModel.cs

```diff
@@ -37,12 +40,14 @@ public class ModifyPaymentViewModel : BaseViewModel
             this.dialogService = dialogService;
             this.paymentManager = paymentManager;
             this.paymentRepository = paymentRepository;
-
-            TargetAccounts = accountRepository.Data;
-            ChargedAccounts = accountRepository.Data;
+            tempCollection = new ObservableCollection<Account>(accountRepository.Data);
```

Comment:
I would create this in the UpdateOtherComboBox method. Since you don't use it anywhere else and only for transfers IMHO we don't need to init this in constructor.